controller/placeholder.js

```diff
@@ -129,6 +125,12 @@ function setup(placeholderService, should_execute) {
           results.filter(layersFilter).map(synthesizeDocs))
         .filter(countryFilter);
 
+        const messageParts = [
+          '[controller:placeholder]',
+          `[result_count:${_.get(res, 'data', []).length}]`
```

Comment:
Since `data` is a direct child of `res` instead of a nested chain of objects, `_.defaultTo` can be used instead of `_.get`.